src/main/java/net/sf/jabref/importer/fileformat/IsiImporter.java

```diff
@@ -73,7 +74,7 @@ public String getId() {
 
     @Override
     public String getDescription() {
-        return null;
+        return "Importer for the ISI Web of Science, INSPEC and Medline format.";
```

Comment:
if its also for inspec and medline, then .txt should also be an extension, shouldn't it?